src/sentry/incidents/logic.py

```diff
@@ -808,6 +809,14 @@ def subscribe_projects_to_alert_rule(alert_rule, projects):
     )
 
 
+def enable_alert_rule(alert_rule):
+    if alert_rule.status == AlertRuleStatus.PENDING.value:
```

Comment:
Should this be `alert_rule.status != AlertRuleStatus.DISABLED.value`?

Reply:
Hmm. Do we want them to be able to enable snapshpts? Or perhaps if we add another status (which could even be `MANUALLY_DISABLED`, `DOWNGRADE_DISABLED`...)?

Reply:
Oh, you might be right that it's better to filter to `PENDING` only so that we can't enable snapshots... I'll leave it up to you if you want to change it

Reply:
Oh nm, I re-read this and it's correct as is